dts/bindings/gpio/nxp,pcal6408a.yaml

```diff
@@ -1,30 +1,14 @@
 # Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2023 SILA Embedded Solutions GmbH
 # SPDX-License-Identifier: Apache-2.0
 
 description: PCAL6408A 8-bit I2C-based I/O expander
 
 compatible: "nxp,pcal6408a"
 
-include: [i2c-device.yaml, gpio-controller.yaml]
+include: ["nxp,pcal64xxa.yaml"]
 
 properties:
   ngpios:
     required: true
     const: 8
```

Comment:
This does not mean what you think it means.

Reply:
Okay, I get it. Hence there would still be a redundancy in the bindings anyway as it is not really possible to derive from a different binding.